### ISSUE TITLE
Micropub endpoint

### ISSUE THREAD

**Issue by metbril** [getgrav/grav#2123]
Feature request: a Micropub endpoint for Grav.

https://indieweb.org/Micropub

https://indieweb.org/Micropub/Servers

I guess this would be on the roadmap once the JSON REST API has been released?

**Comment by Perlkonig** [comment]
It would probably be simpler to implement with such an API, but it should be possible to do now. There's already a [Webmention plugin](https://github.com/Perlkonig/grav-plugin-webmention). I can't see how Grav core would ever support this out of the box (themes have to be compliant with `h-entry`), but plugins should get you pretty close.

**Comment by rhukster** [comment]
I agree, this should be either a plugin, or maybe a theme with some extra plugin functionality to work as expected.
